mlir/tools/rocmlir-lib/CMakeLists.txt

```diff
@@ -184,7 +206,7 @@ if(BUILD_FAT_LIBROCKCOMPILER)
       PATTERN "LICENSE.TXT"
     )
 
-    install(DIRECTORY ${MLIR_INCLUDE_DIRS}
+    rocm_install(DIRECTORY ${MLIR_INCLUDE_DIRS}
       DESTINATION include/${PACKAGE_NAME}/external
       COMPONENT ${LIBRARY_NAME}-headers
```

Comment:
As above, why can this not use the standard `devel` component?

Reply:
@zhanglx13 Any specific reason why we used a separate component here?

Reply:
This is because we provide static libraries for both MIOpen and MIGraphX. These headers are required only by MIGraphX.
I'm not aware of the motivation of using rocm-cmake to create a cmake package of rocMLIR. If this is to make things more standard, maybe we should think about putting everything into the standard `devel` component.